### ISSUE TITLE
Can't delete multiple files from tree view

### ISSUE THREAD

**Issue by arkadiyt** [atom/atom#15286]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

If you select multiple files in tree view and attempt to delete them, it will change the selection to a single file and only delete that 1 file.

### Steps to Reproduce

1. Select multiple files in tree view
2. Right click on files, select 'Delete'

Here's a gif:
![files](https://user-images.githubusercontent.com/2279289/29254030-14eee152-8041-11e7-9631-6782c2d26ccd.gif)

**Expected behavior:**

All the selected files get deleted.

**Actual behavior:**

Only a single file gets deleted.

**Reproduces how often:**

100% reproducible.

### Versions

```
MacBook-Pro:Desktop atetelman$ atom --version
Atom    : 1.19.0
Electron: 1.6.9
Chrome  : 56.0.2924.87
Node    : 7.4.0
MacBook-Pro:Desktop atetelman$ apm --version
apm  1.18.2
npm  3.10.10
node 6.9.5 x64
python 2.7.10
git 2.11.0
MacBook-Pro:Desktop atetelman$ uname -mprsv
Darwin 16.7.0 Darwin Kernel Version 16.7.0: Thu Jun 15 17:36:27 PDT 2017; root:xnu-3789.70.16~2/RELEASE_X86_64 x86_64 i386
MacBook-Pro:Desktop atetelman$
```

**Comment by 50Wliu** [comment]
This has been fixed in Atom 1.20.0, currently in beta.

**Comment by arkadiyt** [comment]
Glad to hear, thanks!
